Automation/SQL/azureDeploy.ps1

```diff
@@ -0,0 +1,43 @@
+# Creates resource group and Azure SQL logical server
+
+param(
+        # Login parameters
+        [string] [Parameter(Mandatory=$true)] $tenantId,          
+        [string] [Parameter(Mandatory=$true)] $applicationId,
+        [string] [Parameter(Mandatory=$true)] $secret,
+        [string] [Parameter(Mandatory=$true)] $subscriptionId,
+        # Azure SQL server parameters
+        [string] [Parameter(Mandatory=$true)] $location,
+        [string] [Parameter(Mandatory=$true)] $resourceGroupName,
+        [string] [Parameter(Mandatory=$true)] $serverName,
+        [string] [Parameter(Mandatory=$true)] $administratorLogin,
+        [SecureString] [Parameter(Mandatory=$true)] $administratorLoginPassword
```

Comment:
change this to String and then convert to securestring so this can all be done with one line of input
`[securestring] $administratorLoginPassword = ConvertTo-SecureString $administratorLoginPassword -AsPlainText -Force `

Reply:
I'll go ahead and change it since that's how I've seen everyone do it, but I read that the point of SecureString is so that the password is never stored as text in memory. If it is stored as a string at any point in the program, there's no real benefit of converting to SecureString later.

Reply:
VS Code is also warning against making this change. Are you sure this should be done?

![image](https://user-images.githubusercontent.com/13280802/107175021-95823180-6980-11eb-996a-90d982a8ba04.png)

Reply:
Yeah so, I think Toddy said that's what we're doing - and it's the only way (I know of) to get everything to run with one line. But yeah, it's not the most secure option. : / @toddysm

Reply:
Changed!